android-kotlin/common/src/main/java/com/google/android/googlevideodiscovery/common/models/VideoEntity.kt

```diff
@@ -6,12 +6,7 @@ import kotlin.time.Duration.Companion.seconds
 
 sealed interface VideoEntity {
     val id: String
-    val duration: Duration
     val name: String
-    val playbackUris: PlatformSpecificUris
-    val images: List<Image>
```

Comment:
nit: we can keep the `images` field as well in the `VideoEntity` interface as all video entities will have images.